soc/wch/ch32v/qingke_v4/vector.S

```diff
@@ -18,12 +18,15 @@ GTEXT(__initialize)
 
 SECTION_FUNC(vectors, ivt)
 	.option norvc
+	/* Jump to 0x08000008, into the main flash zone where j __start is */
+	lui	x5, 0x8000
+	jr	0x8(x5)
```

Comment:
Why doesn't the `j __start` work? Is the linker file correct?

Reply:
no, it jumps where  the j start is, at 0x08000008. The actual issue it is needed with is because the aliased zone contains only the zero-wait flash zone, and when linking for running at 0x08000000, there is also the issue of the offset jumps and generally offset vs absolute adresses which wont always be correct.

In practice, to use non-zero wait flash or run at 0x08000000 instead of at 0x0 in the aliased copy, you need this or it will crash at some point in the code (due to the aforementioned issue).